src/nebula-hooks/use-extended-theme.native.js

```diff
@@ -0,0 +1,29 @@
+import { useTheme, useState, useEffect } from '@nebula.js/stardust';
+
+const useExtendedTheme = () => {
+  const nebulaTheme = useTheme();
+  const [theme, setTheme] = useState(nebulaTheme);
+
+  useEffect(() => {
+    if (nebulaTheme) {
+      const themeProxy = new Proxy(nebulaTheme, {
```

Comment:
@Caele So for the native implementation the index is shift for `getColorPickerColor`. I remember we saw something like this in the client when it was picking the color palette from the default nebula theme and not the sense theme or something?

Reply:
Let's resolve this in another PR.

Reply:
sure